modules/Microsoft.Insights/webTests/deploy.bicep

```diff
@@ -0,0 +1,144 @@
+@description('Required. Name of the webtest.')
+param name string
+
+@description('Required. User defined name if this WebTest.')
+param webTestName string
+
+@description('Optional. Tags of the resource.')
+param tags object = {}
+
+@description('Required. The collection of request properties.')
+param request object
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. User defined description for this WebTest.')
+param descriptionWebTest string = ''
+
+@description('Optional. Unique ID of this WebTest.')
+param syntheticMonitorId string = name
+
+@description('Optional. The kind of WebTest that this web test watches.')
+@allowed([
+  'multistep'
+  'ping'
+  'standard'
+])
+param kind string = 'standard'
+
+@description('Optional. List of where to physically run the tests from to give global coverage for accessibility of your application.')
+param webTestGeolocation array = [
```

Comment:
I'd suggest to rename to `locations` as aligned with the API

Reply:
updated as suggested